src/download.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { verifyStarterExists } from './download';
+
+describe('download', () => {
+  describe('verifyStarterExists', () => {
+    it('returns false if starter does not exist', async () => {
+      expect(await verifyStarterExists({ repo: 'foo/bar' } as any)).toBe(false);
```

Comment:
Minor, could we add the `name` property to the `Starter` object passed to `verifyStarterExists` in these tests? It's the only other required field on the object, and adding that would allow us to drop the type assertion:
```suggestion
      expect(await verifyStarterExists({ starter: 'foo-bar-starter', repo: 'foo/bar' })).toBe(false);
```